docs/api/scheduler.md

```diff
@@ -1151,3 +1151,81 @@ Endpoint to retrieve historical data about the number of total containers by tim
 }
 ```
 
+
+## Endpoint healthcheck
+
+Endpoint to retrieve historical data about critical logs, negative resource on node/cluster/app, ...
```

Comment:
Endpoint to trigger a lightweight internal health check. The health check runs a few validations to verify if the scheduler state is healthy. Such as scanning scheduling errors, detect failed nodes, detect inconsistent resources, etc. The response contains a list of results for each of the checked items.